### PR TITLE
PDQ calculations description

### PR DESCRIPTION
**Pull request by eanders**

### CODE REVIEW

File: app/views/data_quality_reports/version_four/project/README.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+### Bed Inventory
+
+The sum of the `BedInventory` fields for project inventories with inventory covering any
+part of the reporting period.
+
+### Average Daily / Nightly Clients
+
+*NOTE:* Calculated from computed service records.
+
+The sum of the number of service entries on each day of the reporting period divided by
```
Comment (eanders) [inline app/views/data_quality_reports/version_four/project/README.md]: This should note that these exclude extrapolated Street Outreach intentionally.

File: app/views/data_quality_reports/version_four/project/README.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+
+## Completeness
+
+Each attribute is assigned to the first matching bucket in the order:
+refused, not collected, missing, partial, complete.
+
+### Completeness Percentage
+
+Denominator: The number of enrollments * the number of completeness metrics)
+Numerator: Denominator - The number of metrics not marked _complete_.
```
Comment (eanders) [inline app/views/data_quality_reports/version_four/project/README.md]: This line looks odd with `Numerator: Denominator -`

File: app/views/data_quality_reports/version_four/project/README.md [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+### Disabling Condition
+
+Computed for enrollments using `DisablingCondition` if the client is an adult on the report date.
+Disabling Condition is missing if `DisablinghCondition` is blank, or if it is '0', and if any of the most
+recent disability responses are '1'.
+
+### Prior Living Situation
+
+Computed for enrollments where the client is either the head of household, or an adult on
+the report date, using`LivingSituation`.
```
Comment (eanders) [inline app/views/data_quality_reports/version_four/project/README.md]: this is on the entry date to the enrollment right?

Comment (awisema) [reply app/views/data_quality_reports/version_four/project/README.md]: `is_adult?(date: Date.today) || head_of_household`, is this what you are asking?

Comment (eanders) [reply app/views/data_quality_reports/version_four/project/README.md]: sorry, no.  What's `report date`?

Comment (awisema) [reply app/views/data_quality_reports/version_four/project/README.md]: Is 'on the date the report is generated' clearer?

Comment (eanders) [reply app/views/data_quality_reports/version_four/project/README.md]: hmm, I'm starting to question the `Date.today`.  I think that should be the entry date in this situation.

Comment (awisema) [reply app/views/data_quality_reports/version_four/project/README.md]: I agree. To discuss on Monday?

Comment (eanders) [reply app/views/data_quality_reports/version_four/project/README.md]: Monday

File: app/views/data_quality_reports/version_four/project/README.md [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+
+The number of enrollments with `RelationshipToHoH` blank or '1' for the project during the reporting period for entry/exit projects, or with service during the reporting period for night-by-night projects.
+
+### Entering Clients
+
+The number of enrollments for the project during the reporting period with an `EntryDate` during the reporting period.
+
+### Entering Households
+
+The number of enrollments for the project during the reporting period with `RelationshipToHoH` blank or '1', and with an exit record and the `EntryDate` during the reporting period.
```
Comment (eanders) [inline app/views/data_quality_reports/version_four/project/README.md]: Is this correct that it has a requirement to have an exit record?

Comment (awisema) [reply app/views/data_quality_reports/version_four/project/README.md]: No -- copy/paste error.

**Pull request merged**